model/database/common.yaml

```diff
@@ -7,8 +7,9 @@ groups:
       - ref: db.operation.name
         requirement_level:
           conditionally_required: >
-            If readily available. The operation name MAY be parsed from the query text,
-            in which case it SHOULD be the first operation name found in the query.
+            If readily available and if there is a single operation name that describes the
```

Comment:
what we do on the prepared statements case? 
For example, imagine a prepared statement that uses several operations, but since that has a name, that name could be used, which will make it easier to identify the operation. On the Postgres-JS prototype, if a statement has a name, it is using that as the operation name, otherwise tries to grab the first operation name.
My suggestion is to add a comment here, that if the query has a name, it can be used instead, no matter how many operations it has, otherwise follow the rules that you added here

Reply:
That was my goal with the first sentence

> If readily available and if there is a single operation name that describes the database call.

I believe when you execute a prepared statement or stored procedure, you'd write something like `CALL procedure-name` or `EXECUTE stmt_name` or `EXEC  procedure-name`

In this case the would still be one command and one target name and this description seems to work.

But it made me think that  `CALL procedure-name` should result in 

```
db.operation.name = CALL
db.collection.name = procedure-name
```

wonder if you and other folks think about it (we don't have it documented)

Reply:
That was indeed what I wanted to clarify, the result of _operation_ and _collection_ for the store procedure case.
Agree that the operation should be `CALL`, `EXEC` etc, and the collection should be the procedure name. So is worth making that clear on the collection name section

Reply:
Agree! 

If you don't mind, I'd prefer to do it in a separate PR since it's a slightly different concern - created https://github.com/open-telemetry/semantic-conventions/issues/1491

Reply:
I don't mind at all 😄